libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -54,6 +62,27 @@ export class ReadingListEffects implements OnInitEffects {
     )
   );
 
+  markBookAsRead$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ReadingListActions.markBookAsRead),
+      concatMap(({ item }) =>
+        this.http.put(`/api/reading-list/${item.bookId}/finished`,item).pipe(
+          switchMap((data: {finishedDate: string}) =>{
+            const bookUpdate: Update<ReadingListItem | Book> = {
+              id: item.bookId,
+              changes: { finishedDate: data.finishedDate, finished: true }
+            };
+            return [ReadingListActions.confirmedMarkBookAsRead({item: bookUpdate}), BooksActions.markBookAsFinished({book: bookUpdate})]
```

Comment:
Only one action is needed here, I think only `markBookAsFinished` is needed, and reading list reducer can listen on that action from books actions.

Reply:
Addressed review comment